mqt/bench/tests/test_all.py

```diff
@@ -604,3 +599,39 @@ def test_configure_end():
         os.remove(os.path.join(test_qasm_output_path, f))
     os.rmdir(test_qasm_output_path)
     utils.set_qasm_output_path()
+
+
+@pytest.mark.parametrize(
+    "abstraction_level",
+    [
+        (1),
+        (2),
+        (3),
+    ],
+)
+def test_saving_qasm_to_alternative_location_with_alternative_filename(
+    abstraction_level: int,
+):
+    directory = "."
+    filename = "ae_test_qiskit"
+    qc = get_one_benchmark("ae", abstraction_level, 5)
+    assert qc
+    res = qiskit_helper.get_mapped_level(
+        qc, "ibm", qc.num_qubits, "ibm_washington", 1, False, False, directory, filename
+    )
```

Comment:
just a hint for the future (does not necessarily have to be adopted right here and now):
for function calls like these it would really help to use keyword arguments, so that the meaning of each of the parameters is more clear. I can't really tell (from a reviewer perspective) whether the arguments supplied to these functions "make sense".

Reply:
Thanks for that feedback. I will do that in the future.